src/content/getting-started/observe-your-clusters-and-apps/_index.md

```diff
@@ -1,15 +1,107 @@
+
+## Requirements
+
+First step, you need a running workload cluster. If you don't have one, please first [create a workload cluster]({{< relref "/getting-started/provision-your-first-workload-cluster" >}}). Second, you need to deploy the `hello-world` application explained [in previous step]({{< relref "/getting-started/install-an-application" >}}). Further, make sure you have installed [`jq`](https://jqlang.github.io/jq/download/) on your local.
+
+In case you are running your own application, you need to make sure that your application is already [instrumented](https://opentelemetry.io/docs/concepts/instrumentation/) to export metrics.
+
+Also, consider that ingesting new metrics into the platform impacts your costs. The central monitoring system's resource consumption is actually related to the number of metrics it has to handle, so please choose which metrics you want to ingest.
+
+## Step 1: Create a service monitor
+
+In case of running the `hello-world` application, there is a baked-in service monitor that scrapes the metrics from the application. You can enable it by setting `serviceMonitor.enabled` to `true` in the values of the helm chart. If you are running your own application, you need to create a service monitor that scrapes the metrics from your application. This is an example of a service monitor you can use:
+
+```yaml
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    application.giantswarm.io/team: team-rocket
+    app.kubernetes.io/instance: my-service
+  name: my-service
+  namespace: my-namespace
+spec:
+  endpoints:
+  - interval: 60s
+    path: /metrics
+    port: web
+  selector:
+    matchLabels:
+      app.kubernetes.io/instance: my-service
+```
+
+An important bit to notice is the `application.giantswarm.io/team` label. This label is necessary as it's required for the `Prometheus` agent to discover the target. At the same time, it's used as unique identifier for your tenant or team.
+
+Reading the manifest, the application will be scraped every 60 seconds. It will read the metrics from the `/metrics` endpoint using the [port name](https://kubernetes.io/docs/concepts/services-networking/service/#field-spec-ports) as a regular service does. Also, the `app.kubernetes.io/instance` label is used to identify the application, so it should match the application label.
+
+After applying the service monitor, you can open the `Explore` view in the `Grafana` UI and start querying the metrics from your application.
+
+![Explore application metrics](explore-application-metrics.png)
+
+__Note__: The `Grafana` UI is accessible via ingress. To know the address you can check the ingress resource in the `monitoring` namespace of platform API.
```

Comment:
Is this really how we want customers to get the ingress?

Reply:
well it is the way we have now, with Backstage will be easy but it is not there yet

Reply:
I think we have a page explaining how to access grafana. Maybe we can link it here instead?